Header header
explorer_turtle/Marker[] markers
bool[] detected